### ISSUE TITLE
Target class [Tanmuhittin\LaravelGoogleTranslate\Commands\TranslateFilesCommand] does not exist.

### ISSUE THREAD

**Issue by Surt** [barryvdh/laravel-translation-manager#333]
It works on my windows local installation but not in a linux installation:

With laravel 6
```
    "barryvdh/laravel-translation-manager": "dev-master",
```
I did try with v0.5.5, deleting ' laravel bootstrap/cache' folder contents, requiring the tanmuhittin library by my own.... No luck. 

Any clue of what is happening? 

P.D. By the way, I don't really want the google translate functionality, but I didn't found any way to deactivate it (required on the laravel-translation-manager composer.json)

```
Generating autoload files
> Illuminate\Foundation\ComposerScripts::postAutoloadDump
> @php artisan package:discover

In Container.php line 806:
                                                                               
  Target class [Tanmuhittin\LaravelGoogleTranslate\Commands\TranslateFilesCom  
  mand] does not exist.                                                        
                                                                               

In Container.php line 804:
                                                                               
  Class Tanmuhittin\LaravelGoogleTranslate\Commands\TranslateFilesCommand doe  
  s not exist                                                                  
                                                                               

Script @php artisan package:discover handling the post-autoload-dump event returned with error code 1

In Container.php line 806:
                                                                               
  Target class [Tanmuhittin\LaravelGoogleTranslate\Commands\TranslateFilesCom  
  mand] does not exist.                                                        
                                                                               

In Container.php line 804:
                                                                               
  Class Tanmuhittin\LaravelGoogleTranslate\Commands\TranslateFilesCommand doe  
  s not exist                                                                  
                                                                               


In Container.php line 806:
                                                                               
  Target class [Tanmuhittin\LaravelGoogleTranslate\Commands\TranslateFilesCom  
  mand] does not exist.                                                        
                                                                               

In Container.php line 804:
                                                                               
  Class Tanmuhittin\LaravelGoogleTranslate\Commands\TranslateFilesCommand doe  
  s not exist                                     

```

**Comment by Silviobramas** [comment]
I have the same problem. In my local Xampp environment it's working. But not on my a2hosting. Im using PHP 7.3. The problem apperas when I updated from laravel 5.5 to 5.6.
Any idea to solve it?

 ReflectionException  : Class Tanmuhittin\LaravelGoogleTranslate\Commands\TranslateFilesCommand does not exist

  at /home/mfujacob/omp/vendor/laravel/framework/src/Illuminate/Container/Container.php:767
    763|         if ($concrete instanceof Closure) {
    764|             return $concrete($this, $this->getLastParameterOverride());
    765|         }
    766|
  > 767|         $reflector = new ReflectionClass($concrete);
    768|
    769|         // If the type is not instantiable, the developer is attempting to resolve
    770|         // an abstract type such as an Interface of Abstract Class and there is
    771|         // no binding registered for the abstractions so we need to bail out.

  Exception trace:

  1   ReflectionClass::__construct("Tanmuhittin\LaravelGoogleTranslate\Commands\TranslateFilesCommand")
      /home/mfujacob/omp/vendor/laravel/framework/src/Illuminate/Container/Container.php:767

  2   Illuminate\Container\Container::build("Tanmuhittin\LaravelGoogleTranslate\Commands\TranslateFilesCommand")
      /home/mfujacob/omp/vendor/laravel/framework/src/Illuminate/Container/Container.php:646

**Comment by lucasvdh** [comment]
Same issue here.

The `tanmuhittin/laravel-google-translate` requirement was added in `v0.5.4` so I just reverted to `v0.5.3`.

**Comment by tanmuhittin** [comment]
Working on it 👍

**Comment by lucasvdh** [comment]
@tanmuhittin I got this issue on a project which still runs on Laravel 5.5 and required the `dev-master` of the `barryvdh/laravel-translation-manager` package.

However, when looking at the latest version of the `tanmuhittin/laravel-google-translate` package I noticed that the commands folder is all lowercase. The command itself specifies its namespace with an uppercase C and I don't see any reference in the composer.json autoload section. Maybe that's the issue?

**Comment by tanmuhittin** [comment]
@lucasvdh `tanmuhittin/laravel-google-translate` uses psr-4 autoloader. As you mentioned, according to https://www.php-fig.org/psr/psr-4/ 2.3.2, subfolders (in our case commands folder) should be uppercase. So, I published a new release https://github.com/tanmuhittin/laravel-google-translate/releases/tag/1.0.3 which should resolve this issue. But since I cannot reproduce, could you guys please verify if this fix works.

**Comment by lucasvdh** [comment]
@tanmuhittin awesome, I'll try and test it today :+1:

**Comment by lucasvdh** [comment]
@tanmuhittin thanks, tested it and the issue has been resolved in `v1.0.3` :tada:

@barryvdh or @Surt, the issue can be closed.
